contrib/codegen/templates/sets/sets.gotmpl

```diff
@@ -44,58 +45,66 @@ func New{{ $resource.Kind }}Set({{ $kindLowerCamel }}List ...*{{ $import_prefix
     return &{{ $kindLowerCamel }}Set{set: makeGeneric{{ $resource.Kind }}Set({{ $kindLowerCamel }}List)}
 }
 
-func (s {{ $kindLowerCamel }}Set) Keys() sets.String {
+func New{{ $resource.Kind }}SetFromKubeList({{ $kindLowerCamel }}List *{{ $import_prefix }}.{{ $resource.Kind }}List) {{ $resource.Kind }}Set {
```

Comment:
nit: remove `Kube` here as my hope is to support non-kube backends at some point